sdk/eventhub/event-hubs/README.md

```diff
@@ -96,14 +96,32 @@ const partitionIds = await client.getPartitionIds();
 
 ### Publish events to an Event Hub
 
-In order to publish events, you'll need to create an `EventHubProducer`. Producers may be dedicated to a specific partition, or allow the Event Hubs service to decide which partition events should be published to. It is recommended to use automatic routing when the publishing of events needs to be highly available or when event data should be distributed evenly among the partitions. In the our example, we will take advantage of automatic routing.
+In order to publish events, you'll need to create an `EventHubProducer`. Producers may be dedicated to a specific partition, or allow the Event Hubs service to decide which partition events should be published to. It is recommended to use automatic routing when the publishing of events needs to be highly available or when event data should be distributed evenly among the partitions. In the below examples, we will take advantage of automatic routing.
 
-You can also use the [send](https://azure.github.io/azure-sdk-for-js/event-hubs/classes/eventhubproducer.html#send) method to send multiple events using a single call.
+#### Send a single event or an array of events
+
+Use the [send](https://azure.github.io/azure-sdk-for-js/event-hubs/classes/eventhubproducer.html#send) method to send multiple events using a single call.
 
 ```javascript
 const client = new EventHubClient("connectionString", "eventHubName");
 const producer = client.createProducer();
 await producer.send({ body: "my-event-body" });
+await producer.send([{ body: "foo" }, { body: "bar" }]);
+```
+
+#### Send a batch of events
+
+Use the [createBatch](https://azure.github.io/azure-sdk-for-js/event-hubs/classes/eventhubproducer.html#createbatch) method to create
+an `EventDataBatch` object which can then be sent using the [send](https://azure.github.io/azure-sdk-for-js/event-hubs/classes/eventhubproducer.html#send) method.
+Events may be added to the `EventDataBatch` using the [tryAdd](https://azure.github.io/azure-sdk-for-js/event-hubs/classes/eventdatabatch.html#tryadd)
+method until the maximum batch size limit in bytes has been reached.
+
+```javascript
+const client = new EventHubClient("connectionString", "eventHubName");
+const producer = client.createProducer();
+const eventDataBatch = await producer.createBatch();
+eventDataBatch.tryAdd({ body: "my-event-body" });
```

Comment:
Maybe add another line to show that batch can deal with multiple events?
Should we care about the output of tryAdd?

Reply:
I was trying to keep it simple since we have the full sample available, but I can add a variable that holds the output of `tryAdd` so it's clear it returns something instead of throwing